cli/cli/infra.py

```diff
@@ -287,32 +230,42 @@ def delete_namespace(self):
             namespace_id=namespace.id, region=namespace.region
         )
 
-    def _get_container_env_vars(self):
+    def _get_container_env_vars(self) -> tuple[dict[str, str], dict[str, str]]:
         instance = self._get_database_instance()
         if instance is None:
             click.secho("No database found", fg="red", bold="true")
             raise click.Abort()
 
+        endpoint = instance.endpoints[0]
```

Comment:
The `.endpoint` attribute is deprecated in the RDB API: https://www.scaleway.com/en/developers/api/managed-database-postgre-mysql/#path-database-instances-get-a-database-instance